spec/models/conference_spec.rb

```diff
@@ -7,6 +7,22 @@
   it { is_expected.to have_many(:attendees) }
   it { is_expected.to validate_presence_of(:name) }
   it { is_expected.to validate_presence_of(:round) }
+  it { is_expected.to validate_presence_of(:starts_on) }
+  it { is_expected.to validate_presence_of(:ends_on) }
+
+  describe 'it checks the chronological order of dates' do
+    subject { FactoryGirl.build(:conference) }
+    it 'raises an error with an incorrect order of dates' do
+      subject.starts_on = '2017-07-15'
+      subject.ends_on = '2017-07-07'
+      expect(subject).to have(1).error_on(:ends_on)
```

Comment:
Personal preference, but I'm not a huge fan of the `rspec-collection-matchers` gem (I would love to remove it from the Teams App 😉  ) since it also continues to ship matchers from RSpec < v3, such as `errors_on`... In my opinion there's probably a reason why this is no longer in RSpec.

How about just using plain RSpec here?
```ruby
subject.valid?
expect(subject.errors[:ends_on]).not_to be_empty
```
Shouldn't it do the same?